readme.md

```diff
@@ -35,13 +35,15 @@ Renovate was released in 2017 and already is widely used in the developer commun
 
 ## Using Renovate
 
-The easiest way to use Renovate if you are hosted on GitHub.com or GitLab.com is to install the hosted Renovate app.
+The easiest way to use Renovate if you are hosted on GitHub.com or GitLab.com or DevOps (microsoft) is to install the hosted Renovate app.
```

Comment:
I'm not sure of the best way to phrase this, but I think we should differentiate between GitHub.com and GitLab.com, where Renovate is run by us; and on Azure DevOps, where renovate is self-hosted.

Maybe make it a separate sentence? Something like

> The easiest way to use Renovate if you are hosted on GitHub.com or GitLab.com is to install the hosted Renovate app. On Azure DevOps you can install Renovate as an extension from the marketplace.

Reply:
Yes, very good. I'll do the update.